pytest_fixtures/component/satellite_auth.py

```diff
@@ -298,7 +298,7 @@ def enroll_idm_and_configure_external_auth(default_sat):
 def configure_realm():
     """Configure realm"""
     realm = settings.upgrade.vm_domain.upper()
-    run_command(cmd=f'echo "{settings.ipa.keytab}" > /root/freeipa.keytab')
+    run_command(cmd=f'curl {settings.ipa.keytab_url} > /root/freeipa.keytab')
```

Comment:
Optional, if you want to use curl options instead of shell redirects
```suggestion
    run_command(cmd=f'curl -o /root/freeipa.keytab {settings.ipa.keytab_url}')
```